src/terrasoApi/store.ts

```diff
@@ -14,59 +14,38 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
-import { Middleware, combineReducers, configureStore } from '@reduxjs/toolkit';
-import _ from 'lodash/fp';
-import accountReducer from 'terrasoApi/account/accountSlice';
-import groupReducer from 'terrasoApi/group/groupSlice';
-
-import notificationsReducer from 'notifications/notificationsSlice';
+import {
+  TypedUseSelectorHook,
+  useDispatch as reduxUseDispatch,
+  useSelector as reduxUseSelector,
+} from 'react-redux';
+import createStoreFactory, {
+  DispatchFromStoreFactory,
+  StateFromStoreFactory,
+} from 'terrasoApi/shared/store/store';
 
 import gisReducer from 'gis/gisSlice';
+import groupReducer from 'group/groupSlice';
 import userHomeReducer from 'home/homeSlice';
 import landscapeReducer from 'landscape/landscapeSlice';
 import sharedDataReducer from 'sharedData/sharedDataSlice';
 import storyMapReducer from 'storyMap/storyMapSlice';
 import taxonomiesReducer from 'taxonomies/taxonomiesSlice';
 
-const handleAbortMiddleware: Middleware = () => next => action => {
-  if (_.getOr(false, 'meta.aborted', action)) {
-    next({
-      ...action,
-      type: action.type.replace('rejected', 'aborted'),
-    });
-    return;
-  }
-  next(action);
-};
+export type AppState = StateFromStoreFactory<typeof createStore>;
+export type AppDispatch = DispatchFromStoreFactory<typeof createStore>;
 
-const reducer = combineReducers({
-  account: accountReducer,
-  userHome: userHomeReducer,
+const createStore = createStoreFactory({
   group: groupReducer,
+  userHome: userHomeReducer,
   landscape: landscapeReducer,
-  notifications: notificationsReducer,
```

Comment:
Question: Where are the notifications and account reducer defined now?

Reply:
`terrasoApi/shared/shared/store.ts`